source/wp-content/themes/wporg-themes-2024/src/theme-downloads/view.js

```diff
@@ -0,0 +1,97 @@
+/* global google, jQuery */
+
+async function renderChart( element ) {
+	const { theme = '', labelDate, labelValue } = element.dataset;
+	const url = 'https://api.wordpress.org/stats/themes/1.0/downloads.php?slug=' + theme + '&limit=260&callback=?';
+	let downloads;
+	try {
+		downloads = await jQuery.ajax( {
+			dataType: 'jsonp',
+			url: url,
+		} );
+	} catch ( err ) {
+		return;
+	}
+
+	google.charts.setOnLoadCallback( function () {
+		const data = new google.visualization.DataTable();
+		let count = 0;
+
+		data.addColumn( 'string', labelDate );
+		data.addColumn( 'number', labelValue );
+
+		Object.entries( downloads ).forEach( ( [ key, value ] ) => {
```

Comment:
It's unlikely, but if there was no error and the ajax return value is anything other than an object, `Object.entries( downloads )` will result in a TypeError:

![Screenshot 2024-05-13 at 10 56 21 AM](https://github.com/WordPress/wporg-theme-directory/assets/1017872/96b29fd1-1298-478d-9818-dcafd52e1f85)